mdpow/tests/test_automated_dihedral_analysis.py

```diff
@@ -96,17 +101,33 @@ def gen_data(self, SM25_tmp_dir):
                                 'H29', 'H30', 'H31', 'H32', 'H33', 'H34', 'H35'], dtype=object)
 
     # pre-padding dihedral group statistics
-    DG_O1C2N3S4_mean = -0.13089566578887002
-    DG_O1C2N3S4_var = 0.03473127346296412
+    #DG_O1C2N3S4_mean = -0.13089566578887002
+    #DG_O1C2N3S4_var = 0.03473127346296412
```

Comment:
What do the commented out values mean? 

With version control we tend to avoid leaving commented out code in there. It's just ambiguous: either there's a good reason to replace it, then it is replaced. Or not... and then one has to work harder.

What's the justification for keeping old and new visible?

Reply:
- old data from most recently reduced test data set preceding the current, forgot to remove after generating new data
- handled